code/modules/clothing/under/jobs/security.dm

```diff
@@ -242,3 +242,10 @@
 	icon_state = "camopants"
 	cold_protection = LOWER_TORSO
 	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
+
+/obj/item/clothing/under/rank/commissar
+	name = "krieg Battle Uniform"
+	desc = "What was once a full green uniform has been stripped down for survivability on the inhospitable world of catachan, though now it just seems like a power move to wear this on a frozen tundra."
```

Comment:
```suggestion
	desc = "Under the rigorous trenchcoat of an Commissar is his neatly made service uniform, featuring his nametag and medals on their set of collar dress shirt and service pants."
```

Reply:
I don't know if this is for catachan only or for all commissar. If not, don't merge.

We also need undershirt for normal commissar as well, so make one if you can.